README.md

```diff
@@ -94,151 +88,143 @@ Whether or not this is the final proposed set of features is yet undecided.
-then the element is not activatable.
+The intended effect of this configuration is that the subtree is hidden and
+"collapsed" (note the absense of `contain-intrinsic-size` which makes size
+containment use empty size for intrinsic sizing). This is common when content is
+paginated and the developer allows the user to expand certain sections with
+button clicks. In the `render-subtree` case the developer may also listen to the
+activation event and start rendering the subtree when the event targets the
+element in the subtree. This means that find-in-page is able to expand an
+otherwise collapsed section when it finds a match.
 
-Actions that will trigger activation to an element and all of its ancestors, are
-listed in the [cheatsheet](https://github.com/WICG/display-locking/blob/master/cheatsheet.md).
 
-Activation fires a new `rendersubtreeactivation` event when the element is
-activated. The event fires at the beginning of the animation frame, prior to
-requestAnimationFrame callbacks.
 
 ## Alternatives considered
 
-The `display:none` CSS property causes content subtrees not to render. However,
+The `display: none` CSS property causes content subtrees not to render. However,
 there is no mechanism for user-agent features to cause these subtrees to render.
+Additionally, the cost of hiding and showing content cannot be eliminated since
+`display: none` does not preserve the layout state of the subtree.
 
 `visibility: hidden` causes subtrees to not paint, but they still need style and
 layout, as the subtree takes up layout space and descendants may be `visibility:
 visible`. Second, there is no mechanism for user-agent features to cause
-subtrees to render.
-
-`contain: strict` allows the browser to automatically detect subtrees that are
-definitely offscreen, and therefore that don't need to be rendered. However,
-`contain:strict` is not flexible enough to allow for responsive design layouts
-that grow elements to fit their content. (To work around this, content could be
-marked as `contain:strict` when offscreen and then some other value when
-on-screen (this is similar to `rendersubtree`).) Second, `contain:strict` may or
-may not result in rendering work, depending on whether the browser detects the
-content is actually offscreen. Third, it does not support pre-rendering or
-user-agent features in cases when it is not actually rendered to the user in the
-current application view.
+subtrees to render. Note that with sufficient containment and intersection
+observer, the functionality provided by `render-subtree` may be mimicked with
+some exceptions: find-in-page functionality does not work in unrendered content;
+this relies on more browser heuristics to ensure contained invisible content is
+cheap -- `render-subtree` is a stronger signal to the user-agent that work
+should be skipped.
+
+Similar to `visibility: hidden`, `contain: strict` allows the browser to
```

Comment:
I don't think vis:hidden allows that? Descendants of vis:hidden can be vis:visibile

Reply:
Yep, true. I'll add a qualification.